example/mobile/lib/wallet.dart

```diff
@@ -67,4 +67,49 @@ class WalletConnector {
 
     return txId;
   }
+
+  Future<String> signTransactions(SessionStatus session) async {
+    final sender = Address.fromAlgorandAddress(address: session.accounts[0]);
+
+    // Fetch the suggested transaction params
+    final params = await algorand.getSuggestedTransactionParams();
+
+    // Build the transaction
+    final transaction1 = await (PaymentTransactionBuilder()
+          ..sender = sender
+          ..noteText = 'Signed with WalletConnect - 1'
+          ..amount = Algo.toMicroAlgos(0.0001)
+          ..receiver = sender
+          ..suggestedParams = params)
+        .build();
+    final transaction2 = await (PaymentTransactionBuilder()
+          ..sender = sender
+          ..noteText = 'Signed with WalletConnect - 2'
+          ..amount = Algo.toMicroAlgos(0.0002)
+          ..receiver = sender
+          ..suggestedParams = params)
+        .build();
+    AtomicTransfer.group([transaction1, transaction2]);
+
+    // Sign the transaction
+    final tx1Bytes = Encoder.encodeMessagePack(transaction1.toMessagePack());
```

Comment:
Just as a comment, since the latest version you can use:
`transaction.toBytes()`